### DISCUSSION TITLE
How to run this project locally?

### DISCUSSION THREAD

**Discussion by zydeico** [voltgizerz/rest-api-go-firestore#8] [Q&A] [answered]
I cannot find any reference to how to run this project, any idea ? 🤔

**Comment by voltgizerz** [comment, marked as answer]
@zydeico I just updated [README.md](https://github.com/voltgizerz/rest-api-go-firestore?tab=readme-ov-file#how-to-run-project) file on how to run the project. Just follow those steps, and you can run this project on your local machine.

**Reply by zydeico** [reply]
Awesome! Thanks!
